website/mailinglists/services.py

```diff
@@ -86,6 +93,14 @@ def get_automatic_lists():
             ),
             "moderated": True,
         },
+        {
+            "name": "alumni",
```

Comment:
the alias `alumni@thalia.nu` already exists and is used by the alumni committee. I would call the list `oldmembers`